webpack.config.js

```diff
@@ -12,7 +12,7 @@ function getEntry() {
   const mod = JSON.parse(npmListRes);
   const unbundledModule = ['impor', 'uuid',
   // usb-native modules can not be bundled
-  'usb-detection', '@serialport', 'bindings', 'serialport'];
+  'usb-detection', '@serialport', 'bindings', 'serialport', 'debug', 'ms', 'node-gyp-build', 'node-addon-api'];
```

Comment:
It's weird that `node-gyp-build` is required in a production build. I'm pretty sure that package is used at build time to create native binaries, but I don't see why there would be a runtime dependency. That said, if it unblocks the release, I think we can keep it for now. Can you add a comment explaining why these packages were added?

Reply:
I tend to agree, but I attempted to remove it and it broke. I added a comment and it will be in the next commit.